integration-tests/src/test/java/io/quarkiverse/it/cxf/SOAPExceptionInterceptor.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkiverse.it.cxf;
+
+import javax.xml.ws.soap.SOAPFaultException;
+
+import org.apache.cxf.binding.soap.interceptor.SoapPreProtocolOutInterceptor;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.phase.Phase;
+
+/**
+ * Implemenents an CXF interceptor that throws unconditionally an illegal
+ * state exception on handling a message.
+ *
+ * <p>
+ * This interceptor is intented to be used for testing. Configure a
+ * CXF client with this interceptor and simply test whether there is an
+ * effect, i.e. expect a {@link SOAPFaultException} on sending a message.
+ * </p>
+ *
+ * @author wh81752
+ */
+public class SOAPExceptionInterceptor extends AbstractPhaseInterceptor<Message> {
```

Comment:
I'm not sure this class is getting used as intended.  It appears a WebServiceException is thrown in the relevant test case (test_out_interceptor_present), and not a SOAPFaultException.  Kindly fix, as the tests are broken at the moment.

Reply:
Ok, let me re-check this later. It was working fine so far. The idea of this interceptor is just disable any message exchange by  forcing CXF to throw an exception.

Reply:
Works as expected on my side, no broken test locally.